### PR TITLE
Remove isolcpus karg

### PR DESCRIPTION
**Pull request by RamLavi**
This PR is removing the `isolcpus` kernel argument from the VMs image and also from the documentation, as it is no longer needed on RHEL 8,9 [0].

[0] https://access.redhat.com/articles/6126451

### CODE REVIEW

File: docs/Configuring_openshift_virtualization_vm_for_dpdk.md [hunk 1]
```diff
@@ -232,6 +232,12 @@ spec:
 
 Once the VM is deployed and running, there are configurations that need to be set once on the guest. The changes are explained on each subchapter with appropriate snippets. In the end, an example guest configuration summary script is offered.
 
+### Linux grub cmdLine on guest
+The hugepages are specified on the grub cmdLine:
+```bash
+grubby --update-kernel=ALL --args="default_hugepagesz=1GB hugepagesz=1G hugepages=8"
```
Comment (orelmisan) [inline docs/Configuring_openshift_virtualization_vm_for_dpdk.md:238]: I'm not sure that hugepages=8 is correct, but this is not part of this PR.

**Review by orelmisan — APPROVED** [review]
Thank you.

File: docs/Configuring_openshift_virtualization_vm_for_dpdk.md (hunk 1 above)
Comment (RamLavi) [reply docs/Configuring_openshift_virtualization_vm_for_dpdk.md:238]: You are correct. Will need to update in another PR

**Comment by RamLavi** [comment]
e2e test had to be run on a modified Kkubevirt that fixes the SMT Alignement errors.
```
make test/e2e
podman run --rm  \
           --volume /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:Z \
           --volume /home/ralavi/.kube:/root/.kube:Z,ro \
           --workdir /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup \
           -e KUBECONFIG=/root/.kube/kubeconfig \
           -e TEST_CHECKUP_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup:latest \
           -e TEST_NAMESPACE=dpdk-checkup-ns-1 \
           -e NETWORK_ATTACHMENT_DEFINITION_NAME=dpdk-sriovnetwork-ns-1 \
           -e TRAFFIC_GEN_CONTAINER_DISK_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup-traffic-gen:latest \
           -e VM_UNDER_TEST_CONTAINER_DISK_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup-vm:latest \
           docker.io/library/golang:1.20.6 go test ./tests/... -test.v -test.timeout=9h -ginkgo.v -ginkgo.timeout=9h
=== RUN   TestKubevirtDpdkCheckup
Running Suite: KubevirtDpdkCheckup Suite - /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests
==============================================================================================================
Random Seed: 1697112250

Will run 1 of 1 specs
------------------------------
[BeforeSuite] 
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/test_suite_test.go:56
[BeforeSuite] PASSED [0.006 seconds]
------------------------------
Execute the checkup Job should complete successfully
/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/checkup_test.go:83
• [859.427 seconds]
------------------------------

Ran 1 of 1 Specs in 859.434 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestKubevirtDpdkCheckup (859.43s)
PASS
ok  	github.com/kiagnose/kubevirt-dpdk-checkup/tests	859.473s

```
